dts/bindings/i2s/intel,ssp-clk-gating.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Intel SSP clock gating
+
+compatible: "intel,ssp-clk-gating"
```

Comment:
Ok. The existing one is `intel,ssp-dai`, and this one is already being used by each ssp node (`ssp0` to `ssp5`). I'll give it a try, but that means that this will be duplicated for all nodes (and in the code I'll get them from an arbitrary one, such as `ssp0`). I wonder if we don't/shouldn't have "class" properties for all similar nodes.

Reply:
Or are you talking about having an `intel,ssp-settings`, more like @ceolin comment, @MaureenHelm?

Reply:
Possibly, yes. Can you look at syscon as another option?

Reply:
`syscon` seems more related to misc registers, not specific properties of a given hardware version, so I don't really think it helps here.